.github/workflows/app-pipeline.yaml

```
@@ -63,88 +63,117 @@
           mkdir -p .tmp/bin
           go build -o .tmp/bin/license_serializer ./path/to/license_serializer
 
+  build-artifacts:
+    needs: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [1.18.x]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)